### PR TITLE
IDEX-3511 : Remove deprecated class org.eclipse.che.commons.lang.Strings and its usage

### PR DESCRIPTION
**Pull request by rainf0x**
This is pull request issue https://jira.codenvycorp.com/browse/IDEX-3511

This pull request removes deprecated class  org.eclipse.che.commons.lang.Strings and use com.google.common.base.Strings com.google.common.base.Strings  instead of its.

### CODE REVIEW

File: che-core-vfs-impl/src/main/java/org/eclipse/che/vfs/impl/fs/FSMountPoint.java [hunk 1]
```diff
@@ -14,6 +14,7 @@
 import com.google.common.hash.HashFunction;
 import com.google.common.hash.Hashing;
 import com.google.common.io.ByteSource;
+import com.google.common.io.ByteStreams;
```
Comment (skabashnyuk) [inline che-core-vfs-impl/src/main/java/org/eclipse/che/vfs/impl/fs/FSMountPoint.java]: I think its better to remove FSMountPoint from this changes set.

**Comment by skabashnyuk** [comment]
Other ok

File: che-core-vfs-impl/src/main/java/org/eclipse/che/vfs/impl/fs/FSMountPoint.java (hunk 1 above)
Comment (rainf0x) [inline che-core-vfs-impl/src/main/java/org/eclipse/che/vfs/impl/fs/FSMountPoint.java]: It's only import optimization.

**Comment by evoevodin** [comment]
:+1:

**Pull request merged**